pom.xml

```diff
@@ -78,6 +78,10 @@
       <id>repo.jenkins-ci.org</id>
       <url>https://repo.jenkins-ci.org/public/</url>
     </repository>
+    <repository>
+      <id>maven.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/incrementals/</url>
+    </repository>
```

Comment:
```suggestion
```

see https://github.com/jenkinsci/branch-api-plugin/blob/2b1b650338e8ff2daf0f997ee3b231c5cbe25aa8/.mvn/maven.config#L1

Reply:
ok but I would not merge this PR. This PR is just to demostrate what you ask in the stapler repository. This was only for demo purpose.

Reply:
Sure, but assuming the base PR is merged we can just amend this to `@Ignore` the test with a `TODO` comment reminding the maintainer to reënable it once the fix makes it into a suitable `jenkins.version`.

Reply:
You mean I totally remove the changes to pom.xml and add the
`@Ignore("Enable this test when project update jenkins version >= XYZ")` ?

Reply:
Yes, something like that, so that the PR becomes safe to merge at any time.

Reply:
done

Reply:
also restore import packages order